File: api/HIN_API/Models/BatchEmailSender.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace HIN_API.Models
{
    public class BatchEmailSender
    {
        public int? TemplateId { get; set; }
        public List<ContactInformation> Contacts { get; set; }
        public string ContentHtml { get; set; }
        public string Subject { get; set; }
        public List<Group> Groups { get; set; }
    }
}

File: webapp/HIN_WEB/ClientApp/src/app/model/batchemailsender.ts
import { Contactinformation, Group } from "../contactinformation/contactinformation";

export class Batchemailsender {
  public TemplateId: number;
  public Contacts: Array<Contactinformation>;
  public ContentHtml: string;
  public Subject: string;
  public Groups: Array<Group>;
}
